app/controllers/users.js

```diff
@@ -53,5 +54,18 @@ module.exports = {
         logger.error(`DB: ${error.errors[0]}`);
         res.status(500).json({ error: error.errors[0].message });
       });
+  },
+  usersList(req, res, next) {
+    let page = parseInt(req.query.page) || 1;
+    page = page > 0 ? page : 1;
+    const pageSize = 10;
+    User.findAll({ offset: pageSize * (page - 1), limit: pageSize })
```

Comment:
As a good practice, is nice to always return the promises. Here in the controller or any middleware it doesn't makes much difference but once you start to delegate and modularize into another functions and another files, you will be adding a lot of returns and potentialy missing some of them and having some headaches.